apps/pie-microsite/src/assets/styles/components/_tabs.scss

```diff
@@ -1,16 +1,16 @@
 @use '../fozzie-config' as f;
+@use '../variables' as v;
 
 $tabs-link-padding:                f.spacing(d);
 
 .c-tabs {
   z-index: f.zIndex(mid);
   font-weight: f.$font-weight-bold;
-  padding: 0;
 
   ul {
     $tabs-padding: 64px - 16px; // To match grid-template-rows from layout file.
 
-    margin: 0 0 0 $tabs-padding;
+    margin: 0;
```

Comment:
I'm sure there's a good reason here but are we certain we want to remove this? I remember the tabs have a little complexity to their margin/padding behaviour

Reply:
Now everything lines up with the new `c-content-container` div that padding/ margin is all set in the same place so this isn't needed anymore